submissions/adeyemiJuwon/HTMLforms/signup.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Account create</title>
+</head>
+<body>
+  <header>
+    <div>
+        <input type="search" name="search" value="" aria-label="Search" placeholder="Search">
+            <ul>
+                <li>
+                    <a href="login.html">Log in</a>
+                </li>
+                <li>
+                    <a href="signup.html">Sign Up</a>
+                </li>
+                <li>
+                    <a href="contact.html">Contact</a>
+                </li>
+                <li>
+                    <a href="#">Cart</a>
+                </li>
+              </ul>
+    </div>
+    <div>
+        <a href="../introductionToHtml/index.html">
+        <img src="../introductionToHtml/img/mlogo.png" alt="logo">
```

Comment:
Your logo is coming from introToHTML. Delete the image added to htmlForms.

Reply:
Done that already